composer.json

```diff
@@ -6,21 +6,21 @@
         "smarty/smarty" : "~3.1.33",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
-        "php-http/guzzle6-adapter": ">=2.0.0",
+        "php-http/guzzle7-adapter": "0.1.1",
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "laminas/laminas-diactoros" : "~1.8.4",
+        "laminas/laminas-diactoros" : "2.5.0",
```

Comment:
Do all these other libraries need to change to update phpunit?

Reply:
No, there is no need to update other libraries.